lib/if.c

```diff
@@ -413,13 +413,13 @@ struct interface *if_lookup_by_name_all_vrf(const char *name)
 
 struct interface *if_lookup_by_index_all_vrf(ifindex_t ifindex)
```

Comment:
I think the same change can be added in this func " if_lookup_by_name_all_vrf "  also.

Reply:
good catch, I also made the same change there